src/dashboard/src/main/migrations/0073_index_files_runsql.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""MySQL specific indexes for Files
+"""
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_indexes(apps, schema_editor):
+    if schema_editor.connection.vendor != "mysql":
+        return
+
+    schema_editor.execute(
+        "CREATE INDEX Files_transfer_lvrgv3pn_idx ON Files (transferUUID, currentLocation(767));"
```

Comment:
These index names are manually geneated (`{model}_{field}_{random_value}_idx`). They are trying to imitate the naming convention that Django uses, but aren't really the same. They could really be pretty much anything though.